src/main/java/com/scalar/database/transaction/consensuscommit/RecoveryHandler.java

```diff
@@ -51,6 +67,11 @@ public void recover(Selection selection, TransactionResult result) {
     }
   }
 
+  /**
+   * Rollbacks records on {@link DistributedStorage} to the specified {@link Snapshot}
```

Comment:
Records are rolled back by getting info from the snapshot.
This doesn't change the snapshot.

Reply:
Thank you! Updated it somewhat.